src/components/timepicker/timepicker.ts

```diff
@@ -260,6 +273,7 @@ export class MTimepicker extends ModulVue {
 
         } else {
             this.resetPopupTime();
+            this.$emit('input', '');
         }
```

Comment:
Utiliser `@Emit`

Reply:
En fait ici je suis d'accord pour utilisé this.$emit car on veut pas nécessairement le documenter étant donner que C'est pour la gestion du v-model , c'est la seul exception que je verrais 🤓